forge-gui/res/cardsfolder/upcoming/kardum_patron_of_flames.txt

```diff
@@ -0,0 +1,19 @@
+Name:Kardum, Patron of Flames
+ManaCost:2 R R
+Types:Legendary Creature Demon
+PT:4/3
+K:Haste
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, put a flame counter on it, then seek a card with mana value equal to the number of flame counters on it and exile that card face down.
+SVar:TrigCounter:DB$ PutCounter | CounterType$ FLAME | CounterNum$ 1 | ValidCards$ Self | SubAbility$ DBSeek
+SVar:DBSeek:DB$ ChangeZone | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | AtRandom$ True | NoShuffle$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ 1 | ChangeType$ Card.cmcEQX+YouOwn | RememberChanged$ True 
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | TriggerDescription$ When NICKNAME dies, put the cards exiled with it into their owner's hand. At the beginning of the end step of your next turn, discard those cards.
+SVar:TrigReturn:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+ExiledWithSource | RememberChanged$ True | Origin$ Exile | Destination$ Hand | SubAbility$ DBDelay
```

Comment:
is this `RememberChanged` necessary?

Reply:
I think everything is already remembered by line 8, no need to re-remember it

Reply:
Seems like my last comment on this didn't register. But I think you're right, I'm often to zealous with my remembered